neptune/exceptions.py

```diff
@@ -59,9 +116,23 @@ def __init__(self):
         super(NoChannelValue, self).__init__('No channel value provided.')
 
 
-class LibraryNotInstalled(NeptuneException):
+class NeptuneLibraryNotInstalledException(NeptuneException):
     def __init__(self, library):
-        super(LibraryNotInstalled, self).__init__("Library {} is not installed".format(library))
+        message = """
+{h1}     
+----NeptuneLibraryNotInstalledException---------------------------------------------------------------------------------
+{end}
+Looks like library {library} wasn't installed.
+
+To install run:
+    {bash}pip install {library}{end}
+
+You may also want to check the following docs pages:
+    - https://docs.neptune.ai/getting-started/installation/index.html
+    
+{correct}Need help?{end}-> https://docs.neptune.ai/getting-started/getting-help.html
+""".format(**{'library': library, **STYLES})
```

Comment:
I believe, that this is the reason for failing checks - on Python 2.7 this syntax is not supported. That's why this module cannot be parsed and  subsequently imported anywhere.